dataactvalidator/config/sqlrules/a34_appropriations.sql

```diff
@@ -0,0 +1,12 @@
+SELECT
+    approp.row_number,
+    approp.budget_authority_unobligat_fyb,
+    sf.amount as sf_133_amount
```

Comment:
Does it make sense to add the SF-133 line number as part of the column alias to make it more explicit for folks tracking down errors? Something like `sf_133_2490_amount`? Or do you think the rule text itself is enough of a clue?